.github/workflows/build-and-deploy.yml

```diff
@@ -408,3 +408,65 @@ jobs:
           environment: sandbox
           sha: ${{ github.sha }}
           slack-webhook: ${{ secrets.SLACK_WEBHOOK }}
+
+  deploy_domains_infra:
+    name: Deploy Domains Infrastructure
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main' && github.event_name == 'push'
+    concurrency: deploy_production
+    needs: [deploy-aks-sandbox]
+    environment:
+      name: production
+    permissions:
+      id-token: write
+
+    steps:
+      - uses: azure/login@v2
```

Comment:
Is azure login required?

Reply:
removed after adding SKIP_AZURE_LOGGING to ci